### ISSUE TITLE
Custom notification

### ISSUE THREAD

**Issue by frendhisaido** [awr-ggl/chat#6]
## Overview

FCM bawaan tinode walaupun berhasil kirim notifikasi fcm, tapi data notifikasi yang terkirim kosong. Setelah berhasil diisi, ternyata tidak ada data nama user pengirim. 
Solusi judul dan isi pesan notifikasi kosong adalah kita bikin service terpisah khusus untuk olah data notifikasi dan kirim sendiri ke FCM. 
Kenapa bukan custom plugin fcm tinode nya? Karena ⏳ ...
- Their maintainer also mentioned that all the necessary data has already been provided in the payload, normally we should process that in the client app. 
- But we have no time for that, plus, there is a tool designed to solve this problem and implementation is pretty straightforward.
- Enter bull-mq. 

### How it works
Tldr, Notifikasi ditrigger dari tinode.server.push_fcm ke fcm_worker, di fcm_worker ditambahin "New message from `name` in `group_name`", baru dikirim ke FCM. Worker memanfaatkan noco-db open API ke production database. @awr-ggl/backend 

https://github.com/awr-ggl/fcm_worker

### Notes
- Most likely, kita hanya akan jalanin notifikasi, tapi belum sampai deeplink. See why -> 
- Sejauh yg kupahami, deeplink challenging ini karena cara kerja navigasi main navbar (dashboard, chat, tutorial, profile). 
- Ga bisa langsung on tap, navigator push to chat tab. 
- Mesti on tap notif, open navigator push to dashboard, with dashboard arguments yg berisi data payload notif chat.
